agent/agent/build.gradle.kts

```diff
@@ -224,5 +224,7 @@ configurations {
     exclude("com.fasterxml.woodstox", "woodstox-core")
     // temporarily overriding version until latest azure-json is part of azure-core
     resolutionStrategy.force("com.azure:azure-json:1.3.0")
```

Comment:
this reminds me, can you approve and merge https://github.com/microsoft/ApplicationInsights-Java/pull/4065 first?